aiida/plugins/entry_point.py

```diff
@@ -282,8 +282,11 @@ def get_entry_point(group: str, name: str) -> EntryPoint:
     found = eps().select(group=group, name=name)
     if name not in found.names:
         raise MissingEntryPointError(f"Entry point '{name}' not found in group '{group}'")
-    if len(found.names) > 1:
-        raise MultipleEntryPointError(f"Multiple entry points '{name}' found in group '{group}': {found}")
+    if len(found) > 1:
+        # if the entry points have the same *value*, it does not matter which one we load
+        # (needed e.g. to allow installing aiida-core in user space over an existing system-level installation)
+        if len(set(ep.value for ep in found)) != 1:
+            raise MultipleEntryPointError(f"Multiple entry points '{name}' found in group '{group}': {found}")
```

Comment:
Maybe we can just simplify this in a single if-statement?
```suggestion
    # If multiple entry points are found and they have different values we raise, otherwise if they all
    # correspond to the same value, we simply return one of them
    if len(found) > 1 and len(set(ep.value for ep in found)) != 1:
        raise MultipleEntryPointError(f"Multiple entry points '{name}' found in group '{group}': {found}")
```